django/db/backends/oracle/creation.py

```diff
@@ -231,7 +231,8 @@ def _create_test_user(self, cursor, parameters, verbosity, keepdb=False):
                      CREATE TABLE,
                      CREATE SEQUENCE,
                      CREATE PROCEDURE,
-                     CREATE TRIGGER
+                     CREATE TRIGGER,
+                     SELECT_CATALOG_ROLE
```

Comment:
No, it's not possible to add it to just `v$parameter`. It looks like we could use `SELECT ANY DICTIONARY` as an alternative, but it doesn't seem like there's much difference in terms of privileges.

Reply:
Right now I think this is only needed for running the test suite, but if I was to add the system checks mentioned, it would be required for the user that is running migrate. I'm not sure if this is acceptable or not.

Reply:
I think I found another way of doing this, though it's a little painful.

I think we can create a temporary table in the tablespace of the model with a column that is `VARCHAR2(5000)`. Oracle will let us do this regardless of the `max_string_size` setting, so we then need to insert a 5000 character string into it, which will get truncated to 4000 if `max_string_size` is standard, so we can select the Length of the row. If it's 5000, then `max_string_size` is `extended`, if it's 4000, then it's `standard`.

Does this sound reasonable?